docs/credits.rst

```diff
@@ -62,18 +67,22 @@ Core Package Contributors
 * Benjamin Winkel
 * Bernardo Sulzbach
 * Bernie Simon
+* Bhavya Khandelwal
+* bhavyakh
```

Comment:
Are you sure these are not the same person?

Reply:
Fixed